Makefile

```diff
@@ -1,5 +1,6 @@
 SHELL:=/bin/bash
-L0_VERSION?=$(shell git describe --tags)
+L0_VERSION:=$(shell git describe --tags | sed "s/v//")
+RELEASE:=$(shell cat docs-src/docs/releases.md | sed -n 3p | sed 's/0\.10\../'$(L0_VERSION)'/g')
```

Comment:
This should be taken out - it would execute every time we ran `make`, regardless of the target.